phpcs.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ruleset name="drupal">
+    <description>PHP CodeSniffer configuration for Drupal coding standards in Open Social.</description>
+    <file>./modules/custom</file>
+    <file>./modules/social_features</file>
+    <file>./social.profile</file>
+    <file>./social.install</file>
+    <file>./themes/socialbase</file>
+    <file>./themes/socialblue</file>
+    <config name="ignore_errors_on_exit" value="0"/>
+    <config name="ignore_warnings_on_exit" value="0"/>
+    <config name="default_standard" value="Drupal"/>
+
+    <arg name="extensions" value="php,module,inc,install,test,profile,theme" />
+
+    <exclude-pattern>*/node_modules/*</exclude-pattern>
+    <exclude-pattern>*.css</exclude-pattern>
+    <exclude-pattern>*/assets/*</exclude-pattern>
+    <exclude-pattern>*/components/*</exclude-pattern>
+    <exclude-pattern>*social.info.yml</exclude-pattern>
+    <exclude-pattern>*.md</exclude-pattern>
+    <exclude-pattern>PULL_REQUEST_TEMPLATE\.md</exclude-pattern>
+
+    <config name="drupal_core_version" value="8" />
```

Comment:
We'd have to update this to `9` when we get to that right?

Reply:
Yep, but that is in a separate PR which we havent reviewed yet!